scenarioo-client/app/shared/navigation/navigation.controller.js

```diff
@@ -31,6 +31,22 @@ function NavigationController($scope, $location, LocalStorageService, BranchesAn
 
     SelectedBranchAndBuildService.callOnSelectionChange(loadBranchesAndBuildsService);
 
+    $scope.globalSearch = {
```

Comment:
why do we still use $scope here - shouldnt we use `vm` and `controllerAs` notation now?